plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""The entropy calculation implementation."""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """Calculates the byte entropy of input files."""
+
+  NAME = 'entropy'
+  DESCRIPTION = 'Calculates the entropy of input data.'
+
+  def __init__(self):
+    """Initializes the entropy hasher."""
+    super(EntropyHasher, self).__init__()
+    self._counter = collections.Counter()
+    self._length = 0
+
+  @classmethod
+  def GetAttributeName(cls):
+    """The attribute name for the result."""
+    return 'file_entropy'
+
+  def GetStringDigest(self):
+    """Calculates a unicode string containing the entropy value.
```

Comment:
=> Calculates the byte entropy value.

You do not calculate the unicode string.

Reply:
Done